### PR TITLE
docs(repo): update rulesets loading procedure

### PR DESCRIPTION
**Pull request by P0lip**
**Checklist**

- [x] Tests added / updated
- [x] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/guides/2-cli.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ The Spectral CLI supports loading documents as YAML or JSON, and validation of O
 
 ## Using a Ruleset File
 
-If you don't specify a ruleset file with the `--ruleset` parameter, the Spectral CLI will look for a ruleset file called `.spectral.yml`, `.spectral.yaml`, or `.spectral.json` in the current working directory. If no ruleset is specified and no default ruleset file is found, the built-in rulesets will be used.
+If you don't specify a ruleset file with the `--ruleset` parameter, the Spectral CLI will look for a ruleset file called `.spectral.yml`, `.spectral.yaml`, `.spectral.json` or `.spectral.js` in the current working directory.
+Spectral will refuse to lint the document if no ruleset is specified and no default ruleset file is found.
```
Comment (philsturgeon) [inline docs/guides/2-cli.md:48]: If this is meant to be a new paragraph it will need another line.

Comment (P0lip) [reply docs/guides/2-cli.md:48]: We can have a separate paragraph, sure.
```suggestion

Spectral will refuse to lint the document if no ruleset is specified and no default ruleset file is found.
If you want to have a simple basic ruleset for OpenAPI, this command will let you get started:
`echo "extends: spectral:oas" > .spectral.yaml` 
```

@philsturgeon thoughts?

Comment (philsturgeon) [reply docs/guides/2-cli.md:48]: Is this new behavior in Spectral v6? We used to autodetect and apply OAS if we thought it was OAS, but now folks have to specify that for theirselves?

Comment (dijitali) [reply docs/guides/2-cli.md:48]: I just ran up against this change and found this confusing behaviour.  A few other people in https://github.com/stoplightio/spectral/issues/1796 had a similar experience from the sound of things.

Comment (P0lip) [reply docs/guides/2-cli.md:48]: Ooops, I lost sight of these comments.

> Is this new behavior in Spectral v6? We used to autodetect and apply OAS if we thought it was OAS, but now folks have to specify that for theirselves?

Yeah, it's a deliberate change we've made.
Overall, from a technical standpoint it is quite trivial to load some default ruleset - would probably be like a one-liner kind of a change if we were to make it, so totally straightforward to do.
Our goal was to convince users to write their own style guides, but I'm all in for constant reiteration of the standpoint if this happens to be truly annoying - I believe that most of folks do have some kind of a ruleset, so this change is  seamless for them.

Comment (philsturgeon) [reply docs/guides/2-cli.md:48]: Ok, thanks for clarifying.

**Review by mnaumanali94 — COMMENTED** [review]
Screenshot 2021-09-15 at 9 19 44 PM

Lets add a note here too please.

**Review by mnaumanali94 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -31,6 +31,18 @@ yarn global add @stoplight/spectral-cli
...
+Spectral, being a generic YAML/JSON linter, needs a ruleset in order to be able to lint files.
+You can learn more about the rulesets [here](./docs/getting-started/3-rulesets.md).
+
+If you intend to lint an OpenAPI or AsyncAPI document, we have a few predefined rulesets you can extend to get Spectral up and running.
+To reference them, you can run the following command:
+
+```bash
+echo '{\n\t"extends": ["spectral:oas", "spectral:asyncapi"]\n}' > .spectral.json
+```
+
```
Comment (mnaumanali94) [inline README.md:45]: Also add a callout to say that this is a breaking change moving to V6

**Pull request merged**